File: application/web/src/app/testing/test-data/episode-test-data.ts
import { EpisodeDTO } from "../../shared/models/episode-dto";
import { DateUtil } from "../../shared/utils/date-util";

export class EpisodeTestData {
    static steinsGate_episode1(): EpisodeDTO {
        let episodeDTO: EpisodeDTO = new EpisodeDTO();

        episodeDTO.name = "Prologue of the Beginning and Ending";
        episodeDTO.synopsis = "Steins Gate episode 1 synopsis";
        episodeDTO.durationInMinutes = 24;
        episodeDTO.airDate = DateUtil.parse("2011-04-05");

        return episodeDTO;
    }

    static steinsGate_episode2(): EpisodeDTO {
        let episodeDTO: EpisodeDTO = new EpisodeDTO();

        episodeDTO.name = "Paranoia of Time Leaps";
        episodeDTO.synopsis = "Steins Gate episode 2 synopsis";
        episodeDTO.durationInMinutes = 24;
        episodeDTO.airDate = DateUtil.parse("2011-04-12");

        return episodeDTO;
    }
}

File: application/core/src/test/java/com/korogi/core/domain/testdata/EpisodeTestData.java
package com.korogi.core.domain.testdata;

import static com.korogi.core.domain.Episode.newEpisode;

import java.time.LocalDate;
import com.korogi.core.domain.Episode;

public class EpisodeTestData {
    public static Episode.EpisodeBuilder steinsGateEpisode1_notPeristed() {
        return newEpisode()
            .name("Prologue of the Beginning and Ending")
            .synopsis("Prologue of the Beginning and Ending synopsis here")
            .durationInMinutes(24)
            .airDate(LocalDate.of(2011, 4, 5));
    }
}